lib/init.coffee

```diff
@@ -59,34 +60,31 @@ module.exports =
       lintOnFly: true
 
       lint: (textEditor) =>
+        rules = []
         filePath = textEditor.getPath()
         fileText = textEditor.getText()
-        projectConfigPath = @getConfig(filePath)
+        projectConfig = @getConfig(filePath)
 
         if !fileText
           return Promise.resolve([])
 
-        parameters = [filePath]
-
-        if !projectConfigPath || !projectConfigPath.configPath
-          if !@onlyRunWhenConfig
+        if !projectConfig || !projectConfig.configPath
+          if @onlyRunWhenConfig
             atom.notifications.addError 'Pug-lint config not found'
-          return Promise.resolve([])
+            return Promise.resolve([])
```

Comment:
Ah, some linters completely die if they don't have a configuration, or just don't do anything. Others, like `pug-lint` apparently, provide a basic syntax check. I don't use `pug-lint` myself so I didn't know where it fell on that spectrum, only what the existing code was doing 😉.